pkgs/servers/scaphandre/default.nix

```diff
@@ -0,0 +1,62 @@
+{ stdenv
+, lib
+, rustPlatform
+, fetchFromGitHub
+, pkg-config
+, openssl
+, powercap
+, runCommand
+, dieHook
+}:
+
+let
+  scaphandre = rustPlatform.buildRustPackage rec {
```

Comment:
why bothering with a let binding here?

Reply:
ok got it, you are referring to yourself, you can just put `scaphandre` in the function signature, it will be reinjected

Reply:
Done